monai/networks/utils.py

```diff
@@ -615,6 +626,101 @@ def convert_to_torchscript(
     return script_module
 
 
+def convert_to_trt(
+    model: nn.Module,
+    precision: str,
+    inputs_shape: Sequence[Sequence[int]],
+    dynamic_batchsize: Sequence[Sequence[int]],
```

Comment:
I don't quite understand why these 2 args are "sequence in sequence" typehint?

Thanks.

Reply:
This is a sequence in sequence type because there may be more than one inputs to the network, therefore I suppose each input should has an input shape and corresponding batch range.